cmd/tendermint/commands/run_node.go

```diff
@@ -79,8 +81,45 @@ func runNode(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	moduleToLogLevelMap, err := tmflags.ParseLogLevel(config.LogLevel)
```

Comment:
shouldn't we just put all this logic in root? `tendermint node` is prob the only command that really needs it for now, but down the road we may forget to enable if there are other commands that could benefit from fine grained logger control ...